src/utils/github.mjs

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable camelcase */
```

Comment:
Could we put this file to `./scripts`? It is used only for data prefetching.

Reply:
done